### PR TITLE
Update Deepdirect 0.2.5

### PR DESCRIPTION
**Pull request by Jappy0**
fixing the dependencies to specified versions.

Sorry for any inconvenience, thanks so much.

### CODE REVIEW

**Comment by Jappy0** [comment]
@bioconda-bot help needed

**Comment by Jappy0** [comment]
@BiocondaBot please add label

**Comment by Jappy0** [comment]
Hi there,

An issue with the package deepdirect may be caused by numpy version and Tensorflow version 2.4.1. So I want to fix the tensorflow and keras versions to 2.12.0. Thanks so much for helping resolve the conflicts and merge.

Have a good day.

Best regards,

Jappy

**Comment by peterpru** [comment]
> Hi there,
> 
> An issue with the package deepdirect may be caused by numpy version and Tensorflow version 2.4.1. So I want to fix the tensorflow and keras versions to 2.12.0. Thanks so much for helping resolve the conflicts and merge.
> 
> Have a good day.
> 
> Best regards,
> 
> Jappy

Hi, I've fixed the merge conflict. Would you have the ability to test the package locally, to make sure it works as expected?
